taxi/views.py

```diff
@@ -85,6 +107,19 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
```

Comment:
And here as well. It's a good practice to avoid hardcoding values and instead use configuration settings.